k8_cortx_cloud/solution.yaml

```diff
@@ -21,6 +21,15 @@ solution:
     zookeeper: 3.7.0-debian-10-r182
     rancher: rancher/local-path-provisioner:v0.0.20
   common:
+    metallb:
+      svc_name: cortx-metal-lb
+      name: default
+      protocol: layer2
+      addresses:
```

Comment:
Per https://metallb.universe.tf/installation/#installation-with-helm, this should be a YAML list in the MetalLB values files. We do some conversion of that in the deploy-cortx-cloud logic, but are currently limited to 4 IP Addresses based upon the hard-coding here. Is there a specific reason why we didn't include this as an unbound list that would require less manual conversion into the MetalLb values format?